src/userscripts/linkHarmonyReleaseActions.user.js

```diff
@@ -0,0 +1,315 @@
+ * @returns {Promise<any>} - The retrieved or default value.
+ */
+async function getConfigValue(key, defaultValue) {
+  try {
+    let value = await GM_getValue(key, defaultValue);
+    if (typeof defaultValue === "boolean") {
+      value = Boolean(value);
+    }
+    if (typeof value !== typeof defaultValue && defaultValue !== undefined) {
+      console.warn(
+        `Harmony Link Script: Config value for '${key}' has unexpected type. Using default.`
+      );
+      return defaultValue;
+    }
+    return value;
+  } catch (e) {
+    console.error(
+      `Harmony Link Script: Error getting config value for '${key}'. Using default.`,
+      e
+    );
+    return defaultValue;
+  }
+}
+
+/**
+ * Creates the Harmony link anchor (<a>) element with the icon (<img>).
+ * @param {string} mbid - The MusicBrainz Release MBID.
+ * @returns {HTMLAnchorElement|null} The created anchor element or null if mbid is invalid.
+ */
+function createHarmonyLinkElement(mbid) {
+  if (!mbid || typeof mbid !== "string" || !/^[a-f0-9\-]{36}$/.test(mbid)) {
+    console.warn(
+      "Harmony Link Script: Invalid MBID passed to createHarmonyLinkElement:",
+      mbid
+    );
+    return null;
+  }
+
+  const harmonyLink = document.createElement("a");
+  harmonyLink.href = `${DEFAULT_HARMONY_BASE_URL}${mbid}`;
+  harmonyLink.target = "_blank";
+  harmonyLink.rel = "noopener noreferrer";
+  harmonyLink.title = `View Harmony Actions for ${mbid} (opens in new tab)`;
+  harmonyLink.classList.add(CSS_LINK_CLASS);
+
+  const harmonyIcon = document.createElement("img");
+  harmonyIcon.src = DEFAULT_HARMONY_ICON_URL;
+  harmonyIcon.alt = "Harmony Logo";
+  harmonyIcon.classList.add(CSS_ICON_CLASS);
+
+  harmonyLink.appendChild(harmonyIcon);
+  return harmonyLink;
+}
+
+/**
+ * Processes a table containing release links, adding Harmony icons.
+ * @param {string} tableBodySelector - CSS selector for the table body (tbody).
+ * @param {boolean} digitalOnly - If true, only add icons for "Digital Media" releases.
+ */
+function processReleaseTable(tableBodySelector, digitalOnly) {
+  const releaseTableBody = document.querySelector(tableBodySelector);
+  if (!releaseTableBody) {
+    console.error(
+      `Harmony Link Script: Could not find release table body using selector: "${tableBodySelector}".`
+    );
+    return;
+  }
+
+  // Select BDI elements within release links in the second column
+  const releaseTitleBDIElements = releaseTableBody.querySelectorAll(
+    `tr:not(.subh) > td:nth-of-type(2) ${releaseTitleLinkSelector}`
+  );
+  let addedCount = 0;
+
+  releaseTitleBDIElements.forEach((bdiElement) => {
+    const releaseLink = bdiElement.parentElement;
+    if (!releaseLink || releaseLink.tagName !== "A") return;
+
+    const parentRow = releaseLink.closest("tr"); // Get the table row
+    if (!parentRow) return;
+
+    // --- Format Check ---
+    if (digitalOnly) {
+      const formatCell = parentRow.querySelector("td:nth-of-type(4)"); // 4th column is Format
```

Comment:
This check is a bit unreliable and doesn't work in all scenarios. For example, there is a popular [userscript which adds an additional column](https://github.com/murdos/musicbrainz-userscripts#mb_relationship_shortcuts), shifting the format into the fifth column.
Maybe you could determine the position of the "Format" header cell once for every table and use that index here?